vue/src/vue-elements/post-format.vue

```diff
@@ -819,6 +819,7 @@
                 return this.$store.state.generalSettings.available_shorteners;
             },
             isNewUserPro: function () {
+                console.log(0 === this.license && this.$store.state.is_new_user);
```

Comment:
You can remove the `console.log`. 

If you doubt the data type, you can wrap the response in `Boolean( ... )`.

Reply:
@Soare-Robert-Daniel Done! Thank you for the review